x-pack/docs/en/security/authorization/privileges.asciidoc

```diff
@@ -94,10 +94,21 @@ authenticated user. The operations include
 `manage_pipeline`::
 All operations on ingest pipelines.
 
+ifdef::permanently-unreleased-branch[]
+
+`manage_rollup`::
+All rollup operations. Includes legacy rollup operations, such as creating,
```

Comment:
the first usage of legacy! how exciting. I suspect the other APIs will be pending a move to a dedicated legacy section once Rollup Action is released (officially)? Should we have an open docs issue to track this TODO?

Reply:
Woohoo! Though, technically, we already reference "legacy rollup APIs" here: https://www.elastic.co/guide/en/elasticsearch/reference/master/rollup-apis.html#legacy-rollup-apis

I don't know if we'll do a separate section, but I do plan to label the legacy APIs similar to index templates:

<img width="230" alt="Screen Shot 2020-11-25 at 3 45 53 PM" src="https://user-images.githubusercontent.com/40268737/100279900-5b82c100-2f35-11eb-9898-fb6ed62d845d.PNG">

I also plan to add a similar deprecation warning:

<img width="754" alt="Screen Shot 2020-11-25 at 3 46 31 PM" src="https://user-images.githubusercontent.com/40268737/100279938-6b020a00-2f35-11eb-8409-9a12a7a5b5ca.PNG">

I opened https://github.com/elastic/elasticsearch/issues/65515 so we can track ongoing doc efforts. We can discuss doc plans more, but feel free to edit that issue or toss items on there in the meantime.